### ISSUE TITLE
Desync 1.4

### ISSUE THREAD

**Issue by SpearheadUK** [xBrowserSync/App#34]
### Steps to reproduce
1. make changes on first machine
2. open another browser (or on second machine)
3. make a change

### Expected behaviour
after making changes on first machine extension should sync changes and upon opening second browser extension should immediately update bookmark library.

### Actual behaviour
Opening second browser doesnt trigger sync. then, upon making bookmark change, extension reports a desync in bookmarks and reloads all bookmarks from online version (in one instance of this happening i have recieved duplicated bookmark folders, though i havent been able to replicate this). this process undoes change just made and then continues to work as expected. 

### Client app information
**App Platform:** (Chrome, Firefox, iOS, etc) 
Firefox & chrome

**App Platform Version:** 
extension 1.4
firefox 62
chrome 68

#### Other info
```
I have a feeling that the duplication may have happened after switching from chrome to firefox but have yet to confirm.
```

**Comment by nero120** [comment]
Confirmed as a bug, thanks for logging this @SpearheadUK, will get this resolved shortly.

**Comment by nero120** [comment]
Done, I'll get an updated build which includes the fix onto the [beta release](https://github.com/xBrowserSync/App/releases/tag/BETA1) page tonight.

**Comment by nero120** [comment]
Re-opened as this is not fixed in Firefox due to the fact that the onStartup event **must** be bound to synchronously, yet everything in AngularJS is done asynchronously so I have no idea how I can fix this without ripping out AngularJS. Thanks Mozilla. 😡

**Comment by nero120** [comment]
@SpearheadUK I have properly resolved this issue now (commit 06e9d0a72ec8143ebc2cdef404b3c2d5cfbbb244) in both Chrome and Firefox. I've tested both browsers and uploaded the new builds to the [beta release](https://github.com/xBrowserSync/App/releases/tag/BETA1) page.

If you could do a quick test using these builds to check the issue is now resolved for you I'd greatly appreciate it! 😄

**Comment by nero120** [comment]
This is definitely resolved in the [latest BETA release (c)](https://github.com/xBrowserSync/App/releases/tag/BETA1) so I'm closing this issue. Feel free to respond if you're still experiencing the issue in the latest BETA.
